### ISSUE TITLE
Click 8.1.0 removed deprecated function get_os_args used by sphinx-versions

### ISSUE THREAD

**Issue by andy-maier** [Smile-SA/sphinx-versions#14]
From https://github.com/zhmcclient/zhmc-ansible-modules/runs/5803700243?check_suite_focus=true#step:11:1:

```
sphinx-versioning -l docs/source/conf.py build docs/source docs_build
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.10/x64/bin/sphinx-versioning", line 33, in <module>
    sys.exit(load_entry_point('sphinx-versions===1.1.3.post-am1', 'console_scripts', 'sphinx-versioning')())
  File "/opt/hostedtoolcache/Python/3.9.10/x64/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.9.10/x64/lib/python3.9/site-packages/sphinxcontrib/versioning/__main__.py", line 72, in main
    argv = kwargs.pop('args', click.get_os_args())
AttributeError: module 'click' has no attribute 'get_os_args'
```

Environment:
* Python 3.9.10
* Click 8.1.2
* sphinx-versions from my fork

Click 8.1.0 removed the deprecated function get_os_args(): https://github.com/pallets/click/pull/2130

sphinx-versions needs to be updated to change the use of get_os_args() to use sys.args.

**Comment by riebecj** [comment]
I've submitted #16 which should fix this, but I have to wait for maintainers to approve and merge. Until then, you can use my fork which includes the fixes via:
```bash
pip3 install git+https://github.com/riebecj/sphinx-versions.git
```

**Comment by andy-maier** [comment]
@riebecj Thanks much!

We have so far pinned Click to <8 in our projects that use sphinx-versions as a workaround. That was only partly due to this issue here - there are additional issues with Click 8. We need to work through these other issues as well, before we can unpin Click<8.

On the maintainer question, see #13. 

Also, I had already done something for supporting Click 8.10. See my current local fork https://github.com/andy-maier/sphinx-versions/releases/tag/1.1.3.post-am2 . I need to look into your PR #13 to see if anything else needs to be done.
